napari_allencell_annotator/widgets/template_list.py

```diff
@@ -1,5 +1,6 @@
 from typing import Any, List
 
+from PyQt5.QtWidgets import QPushButton
```

Comment:
this need to be imported under qtpy below

Reply:
changed in https://github.com/aics-int/napari-allencell-annotator/pull/92/commits/b7f16e45109a4a54451ebd3c5e8e8e1ae8da3f4b